draft-ietf-tls-external-psk-importer.md

```diff
@@ -138,14 +178,70 @@ EPSKs may be imported for early data use if they are bound to protocol settings
 otherwise be required for early data with normal (ticket-based PSK) resumption. Minimally, that means ALPN,
 QUIC transport settings, etc., must be provisioned alongside these EPSKs.
 
+##Client ID {#client-id-description}
+The `client_id` is an optional field that is required to be unique for each actor that knows the EPSK.
+This is to prevent Selfie-style reflections.
+If two actors use the same `client_id` this defence may not be effective. 
+This is only necessary in scenarios where more than two actors use the same key, including the case where a single agent will complete PSK handshakes as both the client and the server using the same key.
```

Comment:
This is confusing to me. This sentence suggests that including `client_id` is sufficient, whereas the previous sentence seems to contradict it? Does the previous sentence mean to say the following? 

```
If two actors have the same `client_id` this defense may not be effective.
```

(The difference being "have" vs "use")